### PR TITLE
Fix validate script

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Description
A fix to the validate script, should fix the problem of yelling at you when creating new integrations due to supposed name change

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,9 +24,7 @@ jobs:
       - run:
           name: Validate Files and Yaml
           command: |
-              chmod +x ./Tests/yamlvalidator
-              ./Tests/yamlvalidator
```
[the remark this answers is not in the document]
Comment (hirshbergshachar) [reply .circleci/config.yml]: @ronykoz I think this one validates that it's a yaml file, @asafshen ?

[the remark this answers is not in the document]
Comment (ronykoz) [reply .circleci/config.yml]: we should get a scheme validation error if it's not a yaml file, so we should be covered by the python script

[the remark this answers is not in the document]
Comment (asafshen) [reply .circleci/config.yml]: image

yes it basically validates that all integrations, playbooks and scripts can be marshaled to yaml

@ronykoz I think you are right, lets just make sure that scheme fails if it supplied with invalid yaml, and if so - we can remove the yaml validator from the server repo as well

[the remark this answers is not in the document]
Comment (asafshen) [reply .circleci/config.yml]: this is a binary that validates that all playbooks, integrations and scripts can be marshalled as yaml. (see `yaml-validator.go` in the server repo)

@ronykoz I think you are right, lets just make sure that the schema validation fails when received an invalid yaml, and if so, lets also remove the `yaml-validator.go` from the server

[the remark this answers is not in the document]
Comment (ronykoz) [reply .circleci/config.yml]: @asafshen when is the `yaml-validator.go` being used in the server? 
And the script you and @hirshbergshachar created, the `validate_files_structure.py` does fail when given a yaml which is not in the same format as the scheme

[the remark this answers is not in the document]
Comment (asafshen) [reply .circleci/config.yml]: @ronykoz I believe it not used anywhere else, and only created for this purpose, so we can safely delete it

[the remark this answers is not in the document]
Comment (ronykoz) [reply .circleci/config.yml]: @asafshen So i think we should remove it here first, and then we will make sure we don't have any problems working in this format and than we will remove it from the server

[the remark this answers is not in the document]
Comment (asafshen) [reply .circleci/config.yml]: @ronykoz np, though this file is not imported from the server repo in the build process, its just the static binary in this repo

by the way - we should remove the binary file also

[the remark this answers is not in the document]
Comment (ronykoz) [reply .circleci/config.yml]: yes you are right, removed

**Pull request merged**